extensions/amp-analytics/0.1/test/test-visibility-model.js

```diff
@@ -979,6 +979,18 @@ describes.sandboxed('VisibilityModel', {}, () => {
         totalVisibleTime: 1000,
       });
     });
+
+    it('should fire for 100 percent min and max visibility', () => {
+      const vh = new VisibilityModel({
+        visiblePercentageMin: 100,
+        visiblePercentageMax: 100,
+      }, calcVisibility);
+      const eventSpy = vh.eventResolver_ = sandbox.spy();
+      visibility = 1.0;
```

Comment:
can you start with `visibility = 0.99` and check it's not called initially.

Reply:
Done